### ISSUE TITLE
Objects of verbs of expression, knowing, and perception

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/treedown#14]
How should we label objects of verbs of expression, knowing, and perception?

GBI and Lowfat use `o` when they use anything at all, but this is not consistently labelled. Here is the Treedown equivalent:

```
cj >
    v Ἀκούσας 
    cj* δὲ 
    o ὅτι Ἰωάννης παρεδόθη ἀνεχώρησεν εἰς τὴν Γαλιλαίαν.
```

PROIEL uses `COMP` for this, calling it a complement.  Smyth calls these objects of the verbs, they are variously called complements or arguments of such verbs.

Note that we must use terminology consistent with #8, the decision we make here may affect the terminology we agree to in the other issue.

**Comment by jtauber** [comment]
My concern is less what we call it and more what we conflate it with. If we conflate content clause complements with _direct_ objects (as that's specifically what `o` means, not just "object"), what do we lose and what do we gain?

From a verb subcategorisation point of view alone, I think they are worth distinguishing. If we are willing to distinguish O, IO and O2, why conflate content clauses with O? What about verbs that take both a direct object AND a content clause (can't think of a Greek example off the top of my head but think of the English verb _convince_). 

If there are our three options:
- they are complements distinct from objects (and so "objects" is used in a narrower sense)
- "objects" is used in a broader sense of non-subject complements and content clauses like those in the example are just a type of object but NOT the same as a direct object
- content clauses like those in the example are just direct objects without any further distinction made

then I favour them highest to lowest from top to bottom.

**Comment by jonathanrobie** [comment]
I definitely want to hear a good discussion of this among the linguists before choosing one of these options, and I appreciate you laying this choice out so clearly.  From a modeling perspective, I would like a clear choice and a good clear name that says how we are modeling this.

- If we choose to model them as complements distinct from objects, is there a good name for such complements?  I assume we want to distinguish them from predicative complements `pc`.
- If we choose to model them as objects, but not the same as a direct object, is there a good name for this particular kind of object?
- If we choose to model them as direct objects (and I think you have convinced me that this is convenient but wrong, but I am not the linguist in the room), the naming issue is simple, and it's easy for users, but perhaps misleading as well.

**Comment by jtauber** [comment]
Not to keep bringing in English examples, but how do various people analyse the English sentence “Jonathan convinced James (that) treebanking is fun”?

    :s Jonathan
    :v convinced
    :od James
    :?? (that) treebanking is fun

(especially those that consider content clause complements to be direct objects)

Also if anyone has an equivalent Greek example, let's substitute it.

**Comment by jonathanrobie** [comment]
Now that we allow unspecified `o`, one possibility is:

`:s` Jonathan
`:v` convinced
`:od` James
`:o` (that) treebanking is fun

**Comment by jtauber** [comment]
I'm fine with that (in as much as I don't think it's incorrect) but at the same time the issue is if `o` is supposed to mean "it's an object (in the broader sense) but I don't know what subtype yet" (see #22)

**Comment by rkjtan** [comment]
When a clause or multiple clauses are an object (or subject, in the case of passive verbs), I'm OK with object or complement as one level of label, but would also want to distinguish it as a complete predication (with its own S [implied or expressed], P/V, & potentially non-subject arguments, & adjuncts).

:s Jonathan
:v convinced
 James (I would consider James a recipient & thus more like an indirect object than direct object if you define indirect object as more like a recipient)
(that) treebanking is fun (I would consider this content clause as more like a direct object & the clause is a complete predication).

Treebanking is fun is itself S P/V PC

**Comment by jtauber** [comment]
Unlike terms like "patient" and "recipient", terms like "direct object" and "indirect object" are generally used syntactically and therefore require syntactic diagnostic tests. In the case of English, there are clear diagnostic tests for direct object vs indirect object and CGEL has extensive discussion on this as well as why they don't consider the content clauses in examples like this to be direct objects.

However, this just emphasizes why I'd like to substitute a Greek example or two here. The English diagnostics for direct object vs indirect object largely don't apply to Greek and I for one haven't really thought about what they should be for Greek.

My overall feeling is we don't lose anything by distinguishing these content clauses as something different from a prototypical direct object (even if they are similar to direct objects or even a subtype of direct object). And I still find the fact you can't substitute direct objects with content clauses for arbitrary verbs to be a pretty compelling reason for distinguishing them.

**Comment by rkjtan** [comment]
I agree with James that content clauses should be distinguished. I definitely agree also that the English diagnostics for direct objects vs. indirect objects largely don't apply to Greek. I mention "patient" & "recipient" only because it is an unverified hypothesis of mine (except it is the impression I got over time from working in the text) that animate dative objects tend to be more "recipient"-like & animate accusative objects tend to be more "patient"-like. A couple of examples:

Rom. 7.25: αὐτὸς ἐγὼ … δουλεύω νόμῳ θεοῦ … νόμῳ ἁμαρτίας. I take the dative object to involve the “personified" objects as the "recipients" of service.
2 Cor. 5.13: εἴτε γὰρ ἐξέστημεν θεῷ· εἴτε σωφρονοῦμεν ὑμῖν. I take the dative advantage or disadvantage also to involve the objects as the "recipients"/indirectly affected parties (it is thus also indirect-object-like)
When Porter (Idioms, p. 100) indicated that "the semantic feature of relation is still present with the use of the dative case" & Wallace (Greek Grammar, p. 171) observed that "dative objects are usually related to verbs implying personal relation", I think they were noticing something similar to my impression about dative objects & the verbs that take them (when considering only to animate dative object nouns).

I might be happy with simply indicating that we have dative, accusative (also genitive in some cases, BDF, 93–96 (§§169–78) has a list of ten categories of verbs that can apparently be interpreted as taking genitive objects), clausal objects & to indicate that these are non-subject arguments or complements of the verb, while clausal objects are distinguished as containing their own full predication. Whatever other more specific labels one might chose to use, it is clear that these are non-subject arguments or complements of the verb as distinguished from adjuncts.

**Comment by jtauber** [comment]
I should note that I certainly think it's not easy to distinguish complement _role_ in the following two examples:

* I read [Jonathan's blog post]
* I read [that treebanking is fun]

but you need a very particular type of verb to allow this substitution. 

Maybe I'm slowly being convinced that the _roles_ could largely be conflated. I just get so distracted by the subcategorization issue but perhaps that's irrelevant to role :-)

**Comment by 47rooks** [comment]
Found these in Greek. Are they of any help ?

Μαθθαῖον 19·4 ὁ δὲ ἀποκριθεὶς εἶπεν· οὐκ ἀνέγνωτε ὅτι ὁ κτίσας ἀπ᾿ ἀρχῆς ἄρσεν καὶ θῆλυ ἐποίησεν αὐτούς;
In the negative of course but "read ... that" kind of construction.

Perhaps a part of Deut 8:3 from Rahlfs LXX
ινα ἀναγγείλῃ σοι ὅτι οὐκ ἐπ᾿ ἄρτῳ μόνῳ ζήσεται ὁ ἄνθρωπος, ἀλλ᾿ ἐπὶ παντὶ ῥήματι τῷ ἐκπορευομένῳ διὰ στόματος θεοῦ ζήσεται ὁ ἄνθρωπος.

in the vein of convince someone of something.

**Comment by jonathanrobie** [comment]
Erwin Komen suggests that we use "complement" for these.
